src/ml_tooling/utils.py

```diff
@@ -129,16 +138,29 @@ def _create_param_grid(pipe: Pipeline, param_grid: dict) -> ParameterGrid:
     return ParameterGrid(step_dict)
 
 
-def _validate_estimator(model):
+def _validate_estimator(estimator):
```

Comment:
We don't have any unit tests for _validate_estimator. This would probably be a good time to add one.

Reply:
One with a pure estimator, one pipeline with an estimator and one with pipeline without an estiamtor which should raise. Furthermore we should raise if none of the statements are true.

Reply:
Added!